src/wallet/rpcwallet.cpp

```diff
@@ -4555,6 +4555,30 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
             throw JSONRPCError(RPC_INVALID_PARAMETER, string("Invalid parameter, duplicated address: ") + addrStr);
         };
 
+        std::optional<libzcash::UnifiedAddress> ua = std::nullopt;
```

Comment:
`ua` doesn't need to be declared outside the `if` here, as far as I can tell.

Reply:
Oh, I hadn't realized that this was actually already used in the `recipients.push_back` call.